src/config/parser.rs

```diff
@@ -527,7 +530,7 @@ mod tests {
 
     #[test]
     fn semicolon_error() {
-        fail(&toklist!["a"], ParseError::Expected(&[TokType::Semicolon]));
+        fail(&toklist!["a"], ParseError { tok: None, ty: ParseErrorType::Expected(&[TokType::Semicolon]) });
```

Comment:
Specify here (comment) that the expected value of `tok` is `None` as it is an error at EOF.